### PR TITLE
Add epsg to extents

### PR DESCRIPTION
**Pull request by noellehmcheng**
PTAL @RoyThomsonMonash

### CODE REVIEW

File: LoopProjectFile/Extents.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # import netCDF4
 
 
+from pyproj import CRS
```
Comment (lachlangrose) [inline LoopProjectFile/Extents.py:4]: I'm not sure if its a good idea to require pyproj, it means we have another library with gdal as a dependency. @RoyThomsonMonash do you have any comment?

Comment (RoyThomsonMonash) [reply LoopProjectFile/Extents.py:4]: Avoiding importing pyproj is a good idea as you say, the conversion between utm zones and epsg is very simple so is also not needed in this case.

**Review by RoyThomsonMonash — CHANGES_REQUESTED** [review]
Replace the CRS function in the SetExtents with the changes made to GetExtents (using utm_zone and is_southern to manually create the epsg number.  This removes the need for the pyproj import

**Comment by RoyThomsonMonash** [comment]
EPSG is now set explicitly and so this conversion is no longer needed

**Pull request closed without merging**